packages/atlas/src/views/viewer/MemberView/MemberActivity.tsx

```diff
@@ -0,0 +1,203 @@
+      thumnailUri: 'https://atlas-dev.joystream.app/distributor-1/api/v1/assets/21',
+    },
+  ]
+
+  return (
+    <section>
+      {activity.length === 0 ? (
+        <EmptyFallback title="No activity" subtitle="Go out there and explore!" variant="small" />
+      ) : (
+        <LayoutGrid>
+          <GridItem colSpan={{ base: 12, sm: 8 }} rowStart={{ base: 2, sm: 1 }}>
+            <LayoutGrid>
+              {activity.map((activity, i) => (
+                <GridItem key={i} colSpan={{ base: 12 }}>
+                  <ActivityItem
+                    date={activity.date}
+                    type={activity.type}
+                    title={activity.title}
+                    description={activity.description}
+                    thumnailUri={activity.thumnailUri}
+                  />
+                </GridItem>
+              ))}
+            </LayoutGrid>
+          </GridItem>
+          <GridItem colSpan={{ base: 12, sm: 3 }} colStart={{ sm: -4 }}>
+            <Text variant="h500">Overview</Text>
+
+            <OverviewContainer>
+              <OverviewItem>
+                <IconWrapper icon={<SvgActionBuyNow />} size="large" />
+                <OverviewTextContainer>
+                  <Text variant="t100" secondary>
+                    Bought
+                  </Text>
+                  <Text variant="t300">120</Text>
+                </OverviewTextContainer>
+              </OverviewItem>
+              <OverviewItem>
+                <IconWrapper icon={<SvgActionSell />} size="large" />
+                <OverviewTextContainer>
+                  <Text variant="t100" secondary>
+                    Sold
+                  </Text>
+                  <Text variant="t300">80</Text>
+                </OverviewTextContainer>
+              </OverviewItem>
+              <GridRowWrapper>
+                <OverviewItem>
+                  <IconWrapper icon={<SvgActionMint />} size="large" />
+                  <OverviewTextContainer>
+                    <Text variant="t100" secondary>
+                      Created
+                    </Text>
+                    <Text variant="t300">5</Text>
+                  </OverviewTextContainer>
+                </OverviewItem>
+                <OverviewItem>
+                  <IconWrapper icon={<SvgActionBid />} size="large" />
+                  <OverviewTextContainer>
+                    <Text variant="t100" secondary>
+                      Bidding
+                    </Text>
+                    <Text variant="t300">10</Text>
+                  </OverviewTextContainer>
+                </OverviewItem>
+              </GridRowWrapper>
+            </OverviewContainer>
+          </GridItem>
+        </LayoutGrid>
+      )}
+    </section>
+  )
+}
+
+type ActivityItemProps = {
+  date: string
+  type: string
+  title: string
+  description: string
+  thumnailUri: string
+}
+const ActivityItem: React.FC<ActivityItemProps> = ({ date, type, title, description, thumnailUri }) => {
```

Comment:
One last thing. Maybe we could add this particular component to the storybook? It will be easier to test it out.